MAUI/Expander/Appearance.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: Appearance in .NET MAUI Expander control | Syncfusion
+description: Learn here all about Appearance support in Syncfusion .NET MAUI Expander (SfExpander) control and more.
+platform: MAUI
+control: SfExpander
+documentation: ug
+---
+
+# Appearance in .NET MAUI Expander (SfExpander)
+
+The Expander allows to customize appearance of the Icon, and provides various functionalities to the users.
+
+## Header icon position 
+
+The `SfExpander` allows you to customize the position of the header icon by using the `HeaderIconPosition` property. By default, the header icon position is `Start`. 
+
+{% tabs %}
+{% highlight xaml %}
+    <syncfusion:SfExpander x:Name="expander" HeaderIconPosition="End" />       
+{% endhighlight %}
+{% highlight c# %}
+    expander.HeaderIconPosition = Syncfusion.Maui.Expander.IconPosition.End;
+{% endhighlight %}
+{% endtabs %}
+
+## Header background color customization
+
+The `SfExpander` allows you to customize the background color of the expander header by using the `HeaderBackgroundColor` property.
+
+{% tabs %}
+{% highlight xaml %}
+    <syncfusion:SfExpander x:Name="expander" HeaderBackground="Pink"/>
+{% endhighlight %}
+{% highlight c# %}
+    expander.HeaderBackground = Colors.Pink;
+{% endhighlight %}
+{% endtabs %}
+
+## Icon color customization
+
+The `SfExpander` allows you to customize the color of the expander icon by using the `IconColor` property. By default, the icon color is `Black`.
```

Comment:
The `SfExpander` allows you to customize the color of the expander icon by using the `IconColor` property. By default, the icon color is `Black.`

Reply:
Addressed the concerns provided.